tekton/cronjobs/tekton-service-cd-cron-base/trigger-resource-cd.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2019 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: tekton-release-cd-trigger
+spec:
+  schedule: "12 * * * *" # Houly at *:12
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          volumes:
+          - name: workspace
+            emptyDir: {}
+          containers:
+          - name: trigger
+            image: curlimages/curl
+            command:
+              - /bin/sh
+            args:
+              - -ce
+              - |
+                cat <<EOF > /workspace/post-body.json
+                {
+                  "namespace": "$NAMESPACE",
+                  "cluster-resource": "$CLUSTER_RESOURCE",
+                  "tekton": {
+                    "project": "$TEKTON_PROJECT",
+                    "version": "$TEKTON_VERSION",
+                    "environment": "$TEKTON_CLUSTER",
+                    "bucket": "$RELEASE_BUCKET",
+                    "file": "$RELEASE_FILE"
+                  },
+                  "plumbing": {
+                    "repository": "$PLUMBING_REPOSITORY",
+                    "revision": "$PLUMBING_REVISION"
+                  }
+                }
+                EOF
+                curl -d @/workspace/post-body.json $SINK_URL
```

Comment:
itd be nice to be able to invoke trigger templates without a sink! - maybe something like https://github.com/tektoncd/triggers/issues/200 one day :D

Reply:
> my main feedback is wondering what minio is being used for, and how it works (looks like it uses s3?)
> 
> apologies for the verbose feedback - i just want to try to make sure that it's as easy as possible for as many of us as possible to be able to jump in and out of this project (same as with the other projects) 🙏

That's a very good point, thank you.
I appreciate all the time reviewing this poorly documented work :)